docs/vspec.md

```diff
@@ -22,6 +22,7 @@ vspec export --help
 │ jsonschema                        Export as a jsonschema.                                              │
 │ protobuf                          Export as protobuf.                                                  │
 │ yaml                              Export as YAML.                                                      │
+│ tree                              Export as Tree.                                                      │
```

Comment:
Do we possibly need to say somewhere something about the "Tree export format", like saying that it is a text file and giving  a simple example. Like creating a `tree.md` file and reference it from somewhere in this file.

Reply:
Done